image.md

```diff
@@ -0,0 +1,140 @@
+# Image
+
+This section defines a format for encoding a container's *filesytem bundle* as a portable *image*.
+An image is a set of files organized in a certain way, and containing all of the necessary data and metadata, for any compliant runtime to verify the validity of the image's contents and to construct a filesystem bundle represented by the image.
+Images are meant to be portable such that they can be used to reliably move filesystem bundles between instances of OCI compliant implementations.
+
+## Notational Conventions
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this and other documents of the specification are to be interpreted as described in [RFC2119](http://tools.ietf.org/html/rfc2119).
+
+## Image Contents
+
+A compliant image contains the following pieces of information:
+
+1. An OCI [`config.json`](config.md) file.
+This file is REQUIRED to be present in the image.
+The `config.json` file contains the host independent configuration information necessary to execute the container.
+See [`config.json`](config.md) for more information.
+
+1. The root filesystem of the filesystem bundle.
+This directory MUST be present in the image, however it MAY be empty (not contain any files or folders).
+As specified in the `config.json` file, this directory contains the files that make up the root filesystem of the container.
+This directory MAY contain only a portion of the complete set of files that make up the filesystem bundle, in which case the remaining set of files MUST be found in one of the dependent images.
+
+1. References to dependent images.
+If present, this OPTIONAL information MUST be in a file called `required.json`.
+This file MUST be a json encoded array of references to images as specified in the [Dependent Images](#dependent-images) section.
+
+1. Dependent images.
+An image MAY choose to include dependent images as directories within the image.
+If present, each dependent image MUST be placed in a directory whose name is the same as its ID.
+Within each directory MUST be a complete OCI compliant image definition.
```

Comment:
Ah, the contents are *images* (this file), not bundles.  So I'm removing my earlier comment.